src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToArrayTransformer.php

```diff
@@ -108,7 +109,7 @@ public function transform($dateTime)
      *
      * @param array $value Localized date
      *
-     * @return \DateTime Normalized date
+     * @return \DateTimeInterface Normalized date
```

Comment:
I think leaving it as `\DateTimeImmutable` is still semantically correct, and is more concise